R/download_zenodo.R

```diff
@@ -48,10 +51,11 @@
 #' }
 download_zenodo <- function(doi,
                             path = ".",
-                            parallel = FALSE) {
+                            parallel = FALSE,
+                            quiet = FALSE) {
 
   assert_that(is.string(doi), is.string(path))
-  assert_that(is.flag(parallel))
+  assert_that(is.flag(parallel), is.flag(quiet))
```

Comment:
```suggestion
  assert_that(is.flag(parallel), noNA(parallel), is.flag(quiet), noNA(quiet))
```

Note that `NA` is a 'flag' (a logical of length 1).

Reply:
You should import `noNA` too.

Reply:
Great tip, thanks.